student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] In this section, you called the `super` with arguments, yet, there are no parameters i the `initialize` method above. 
- [ ] The problem here is that these three variables `(name, age, and parent_permission)` are not defined within the initialize method or as parameters of the Student class, so calling `super(name, age, parent_permission)` will result in an error. This [content](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) will help to revise 👍